pkgs/development/tools/omnisharp-roslyn/updater.sh

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p curl jq common-updater-scripts nuget-to-nix
+# shellcheck shell=bash
+
+# WARNING: you need BOTH .NET 7 and 6 to run this script (and they must be on your path
```

Comment:
Can this be handled by the shebang?

Reply:
It cannot because .NET 7 isn't in nixpkgs and I do not know of a way to do `dotnetCorePackages.combinePackages` in the shebang